packages/prisma/migrations/20220316055019_added_username_premium_boolean_to_user_model/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "premiumUsername" BOOLEAN DEFAULT false;
```

Comment:
Let's put this inside the metadata json field since it's not a core feature for product.